src/main/kotlin/io/quarkus/code/misc/QuarkusExtensionUtils.kt

```diff
@@ -17,6 +20,12 @@ object QuarkusExtensionUtils {
     private const val hashMaxLength = 3
     private val maxHashCode: Int
     private val defaultExtensions = setOf("io.quarkus:quarkus-resteasy")
+    private val stopWords = setOf("the", "of", "to", "and", "a", "in", "is", "it", "you", "that", "he", "was",
+            "for", "on", "are", "with", "as", "I", "his", "they", "be", "at", "one", "have", "this", "from", "or",
+            "had", "by", "not", "but", "what", "we", "can", "out", "other", "were", "all", "there", "when", "up",
+            "your", "how", "an", "each", "she", "which", "do", "their", "if", "will", "way", "about", "many", "then",
+            "them", "would", "so", "these", "her", "him", "has", "my", "over", "than", "who", "may", "down", "been")
+    private val tokenizerPattern = Pattern.compile("\\w+");
```

Comment:
I think `StringTokenizer` is more efficient no?

Reply:
I tend to prefer StringTokenizer in situations where the set of delimiter characters is well defined and small. But here, given that description is free text, figuring out all possible delimiter characters is all but straightforward and the list might become rather long. Thus I chose the regular expression.

Reply:
👍